src/group_theory/group_action/basic.lean

```diff
@@ -192,6 +192,8 @@ local attribute [instance] orbit_rel
 
 variables {α} {β}
 
+lemma orbit_rel_rel_iff {x y : β} : (orbit_rel α β).rel x y ↔ x ∈ orbit α y := iff.rfl
```

Comment:
`orbit_rel_apply`? But to make this name work we would need `setoid α` to be `fun_like`. Why isn't it already?

Reply:
To be fair, you don't to refactor anything to add the instance. And you can make a simp lemma `coe_eq_rel : ⇑s = s.rel` to avoid needing to translate the API.